src/main.rs

```diff
@@ -213,29 +212,45 @@ async fn reload_network(auth: BearerAuth) -> impl Responder {
     }
 }
 
-async fn post(path: &str, auth: Option<BearerAuth>) -> Result<HttpResponse> {
-    if let Some(auth) = auth {
-        if !verify_token(auth)? {
-            error!("post {path} verify false");
-            return Ok(HttpResponse::build(StatusCode::UNAUTHORIZED).finish());
-        }
-    }
-
-    let mut sender = match sender().await {
+async fn post_with_json_body(path: &str, body: impl Serialize) -> Result<HttpResponse> {
+    let json = match serde_json::to_value(body) {
+        Ok(r) => r,
         Err(e) => {
-            error!("error creating request sender: {e}. socket might be broken. exit application");
-            std::process::exit(1)
+            error!("Failed to serialize data error: {e:#}");
```

Comment:
afaik we always start error msgs with small letters

Reply:
Updated